jekyll/_cci2/rerun-failed-tests-only.adoc

```diff
@@ -246,6 +246,23 @@ The snippet above will write the list of test file names to `files.txt`.  On a n
 == Known limitations
 
 * If your testing job is using parallelism and test splitting, the job will spin up the number of containers/virtual machines (VMs) that are specified with the `parallelism` key. However, the step that runs tests for each of those parallel containers/VMs will only run a subset of tests, or no tests, after the tests are split across the total number of parallel containers/VMs. For example, if parallelism is set to eight, there may only be enough tests after the test splitting occurs to "fill" the first parallel container/VM. The remaining seven containers/VMs will still start up, but they will not run any tests when they get to the test execution step.  **The vast majority of customers still observe substantial time and credit savings** despite spinning up containers/VMs that do not run tests.  
```

Comment:
```suggestion
* If your testing job uses parallelism and test splitting, the job will spin up the number of containers/virtual machines (VMs) that are specified with the `parallelism` key. However, the step that runs tests for each of those parallel containers/VMs will only run a subset of tests, or no tests, after the tests are split across the total number of parallel containers/VMs.
+
For example, if parallelism is set to eight, there may only be enough tests after the test splitting occurs to "fill" the first parallel container/VM. The remaining seven containers/VMs will still start up, but they will not run any tests when they get to the test execution step. 
+
**The vast majority of customers still observe substantial time and credit savings** despite spinning up containers/VMs that do not run tests.  
```
Adding this suggestion for readability

I'd also suggest phrasing it as "You can still observe substantial..." as we usually try and refer to the reader directly/in second person